controllers/comments.controller.js

```diff
@@ -0,0 +1,16 @@
+const { checkArticleExists } = require("../models/article.model");
+const { articleComments } = require("../models/comment.model");
+
+exports.getArticleComments = (req, res, next) => {
+  const { article_id } = req.params;
+  const promises = [articleComments(article_id)];
+
+  if (article_id) {
```

Comment:
This code block is unnecessary, article_id will exist if you've reached this controller.